topics/genome-annotation/tutorials/tnseq/tutorial.md

```diff
@@ -0,0 +1,681 @@
+---
+layout: tutorial_hands_on
+
+title: "Essential genes detection with Transposon insertion sequencing"
+zenodo_link: "https://doi.org/10.5281/zenodo.940733"
+tags:
+  - bacteria
+  - tnseq
+  - essential genes
+questions:
+  - "What is Transposon insertion Sequencing?"
+  - "How to get TA sites Coverage ? "
+  - "How to predict essential genes ?"
+objectives:
+  - "Understand the read structure of TnSeq analyses"
+  - "Predict Essential genes with Transit"
+  - "Compare gene essentiality in control sample and an different experimental condition"
+time_estimation: "7h"
+key_points:
+contributors:
+  - dlariviere
```

Comment:
It should be `delphine-l` here (as in the `CONTRIBUTORS.yaml` file)

Reply:
I managed to make mistakes on my own name tag 🙄 Thanks !